test/web-platform-tests/run-to-upstream-web-platform-test.js

```diff
@@ -4,64 +4,88 @@ const http = require("http");
 const path = require("path");
 const fs = require("fs");
 const st = require("st");
+const specify = require("mocha-sugar-free").specify;
+
+const nodeResolverPromise = require("../util").nodeResolverPromise;
+const inBrowserContext = require("../util").inBrowserContext;
 const jsdom = require("../..");
 
-function createJsdom(urlPrefix, testPath, t) {
+const globalPool = { maxSockets: 6 };
```

Comment:
Not sure, it is there because I copied over the code from `run-web-platform-test.js` for consistencies sake. I will make sure this part is reused between `run-to-upstream-web-platform-test.js` and `run-web-platform-test.js`